pennylane_lightning/lightning_tensor/_tensornet.py

```diff
@@ -223,20 +221,35 @@ def _preprocess_state_vector(self, state, device_wires):
         if len(device_wires) == self._num_wires and Wires(sorted(device_wires)) == device_wires:
             return np.reshape(state, output_shape).ravel(order="C")
 
-        # generate basis states on subset of qubits via the cartesian product
-        basis_states = np.array(list(product([0, 1], repeat=len(device_wires))))
+        local_dev_wires = np.array(device_wires.tolist())[::-1]
```

Comment:
Also, if the device wires are not passed in as integers, but strings/labels, do we have an issue with this?

Reply:
No problem because the `dev_wires` is mapped before to reach this point